phpunit.xml

```diff
@@ -34,6 +34,7 @@
     <testsuite name="Pim Tests">
       <directory>symfony/plugins/orangehrmPimPlugin/test/Service</directory>
       <directory>symfony/plugins/orangehrmPimPlugin/test/Dao</directory>
+      <directory>symfony/plugins/orangehrmPimPlugin/test/Api</directory>
```

Comment:
I think you can simplify this 
`symfony/plugins/orangehrmPimPlugin/test/*`

Reply:
yes. once migrate old tests, we can go with this way.